packages/ckeditor5-build-balloon-block/build/ckeditor.d.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * @license Copyright (c) 2003-2023, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+import BalloonEditorBase from '@ckeditor/ckeditor5-editor-balloon/src/ballooneditor';
+import '../theme/theme.css';
```

Comment:
I do not understand why it lands in typings. It points to a non-existing file, as builds do not expose anything except for a single entry file.

When using the setup from my comment (https://github.com/ckeditor/ckeditor5/pull/13463#issuecomment-1433067922), everything looks good.

Reply:
I noticed the same, but it actually points to an existing file:

https://github.com/ckeditor/ckeditor5/blob/master/packages/ckeditor5-build-balloon-block/src/ckeditor.js#L37

I will look into ways to remove this, if it's possible at all.

Reply:
Right, I was checking classic build.